content/docs/user-guide/merge-conflicts.md

```diff
@@ -103,27 +103,22 @@ To resolve conflicted `.dvc` files generated by `dvc import` or
 `dvc import-url`, remove the conflicted hashes altogether:
 
 ```yaml
-< < < < < < < HEAD
-md5: 263395583f35403c8e0b1b94b30bea32
-=======
-md5: 520d2602f440d13372435d91d3bfa176
-> > > > > > > branch
 frozen: true
 deps:
 - path: get-started/data.xml
   repo:
     url: https://github.com/iterative/dataset-registry
 < < < < < < < HEAD
     rev_lock: f31f5c4cdae787b4bdeb97a717687d44667d9e62
-=======
+= = = = = = =
```

Comment:
they should be w/o spaces - https://wincent.com/wiki/Understanding_Git_conflict_markers ?

Reply:
The problem is that most IDEs will think it's an actual conflict if we save it like that. Maybe even GitHub or some code analyzers will pick it up as a problem.

Reply:
And we can't use HTML encoding since it's in a md code block.